scripts/zones/Bastok_Markets_[S]/npcs/Kevan.lua

```diff
@@ -0,0 +1,46 @@
+-----------------------------------
+-- Area: Bastok Markets [S]
+--  NPC: Kevan
+-- !pos -308.590 -012.000 -094.227 189
+-- Sealed Container NPC
+-----------------------------------
+local ID = require("scripts/zones/Bastok_Markets_[S]/IDs")
+-----------------------------------
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+    local itemID = trade:getItemId()
+    local itemKeyItemMapping =
+    {
+        [2686] = xi.ki.THE_WORDS_OF_DONHU_I,
```

Comment:
All of these raw item ids need to be added to items.lua and then the full table name used here

Reply:
Done.